src/main/java/org/ilgcc/app/submission/conditions/DisplayContactProvidersPhoneNumberScreen.java

```diff
@@ -0,0 +1,23 @@
+package org.ilgcc.app.submission.conditions;
+
+import formflow.library.config.submission.Condition;
+import formflow.library.data.Submission;
+import java.util.Map;
+import org.ilgcc.app.utils.SubmissionUtilities;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DisplayContactProvidersPhoneNumberScreen extends EnableMultipleProviders implements Condition {
+
+    @Override
+    public Boolean run(Submission submission, String uuid) {
+        Map<String, Object> inputData = submission.getInputData();
+        return super.run(submission) && SubmissionUtilities.isSelectedAsProviderContactMethod(inputData, "TEXT")
+                && isMissingProviderEmail(inputData);
+    }
+
+    private Boolean isMissingProviderEmail(Map<String, Object> inputData) {
```

Comment:
```suggestion
    private Boolean isMissingProviderPhone(Map<String, Object> inputData) {
```
Should `isMissingProviderEmail` be renamed?

Reply:
Good catch. Fixed.